Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM ubuntu:trusty
+MAINTAINER Paul Bowsher <paul.bowsher@gmail.com>
+
+RUN apt-get update
+RUN apt-get install -y sqlite3 libsqlite3-dev git build-essential
+RUN apt-get install -y ruby-dev libssl-dev
+RUN apt-get install -y nodejs
```

Comment:
These RUN statements should probably be combined to reduce the number of layers produced and keep image-size down.